client/src/main/java/com/flipkart/flux/client/config/FluxClientConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package com.flipkart.flux.client.config;
+
+public class FluxClientConfiguration {
+
+  private String url;
```

Comment:
pls keep "http://localhost:9998" as default value, so that to tryout in local client doesn't need to provide a config class.